extensions/interactions/LogicProof/static/js/shared.js

```diff
@@ -559,10 +559,9 @@ var logicProofShared = (function() {
    *         to what the user intended.
    */
   var assignTypesToExpression = function(untypedExpression, possibleTopTypes,
```

Comment:
Generally, if contents of parens span more than a line, break after '('. Ditto below.

Reply:
Done.